Step: build-step-credential-initializer-qbtzx:

{"level":"warn","ts":1556773042.7782466,"logger":"fallback-logger","caller":"logging/config.go:65","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/backport-container-name-fix\" is not a valid GitHub commit ID"}
{"level":"info","ts":1556773042.7789428,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}
Step: build-step-git-source-*********-test-qs-master-msvf6:

{"level":"warn","ts":1556773044.190619,"logger":"fallback-logger","caller":"logging/config.go:65","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/backport-container-name-fix\" is not a valid GitHub commit ID"}
{"level":"info","ts":1556773045.337468,"logger":"fallback-logger","caller":"git-init/main.go:99","msg":"Successfully cloned \"https://github.com/*********/test-qs\" @ \"v0.0.19\" in path \"/workspace/source\""}
Step: build-step-place-tools:

Step: build-step-git-merge:

Warning failed to load version: failed to parse version -dev+9e73adc0b: No Major.Minor.Patch elements found
time="2019-05-02T04:57:27Z" level=info msg="got "
Using SHAs from PULL_REFS=master:
time="2019-05-02T04:57:27Z" level=warning msg="no SHAs to merge, falling back to initial cloned commit"
Step: build-step-setup-**-git-credentials:

Generated Git credentials file /workspace/xdg_config/git/credentials
Step: build-step-build-npm-install:

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN REPLACE_ME@1.0.0 No repository field.
npm WARN REPLACE_ME@1.0.0 No license field.

up to date in 0.36s
found 0 vulnerabilities

Step: build-step-build-npm-test:


> REPLACE_ME@1.0.0 test /workspace/source
> echo "no test specified" && exit 0

no test specified
Step: build-step-build-container-build:

[36mINFO[0m[0000] Downloading base image node:9-slim           
ERROR: logging before flag.Parse: E0502 04:57:36.755039      15 metadata.go:142] while reading 'google-dockercfg' metadata: http status code: 404 while fetching url http://metadata.google.internal./computeMetadata/v1/instance/attributes/google-dockercfg
ERROR: logging before flag.Parse: E0502 04:57:36.758023      15 metadata.go:159] while reading 'google-dockercfg-url' metadata: http status code: 404 while fetching url http://metadata.google.internal./computeMetadata/v1/instance/attributes/google-dockercfg-url
2019/05/02 04:57:36 No matching credentials were found, falling back on anonymous
[36mINFO[0m[0002] Error while retrieving image from cache: getting file info: stat /workspace/sha256:0e0175bbe4f8d9d104590d8d2c591017874e30977cb71aaff1cf6fa99479575c: no such file or directory 
[36mINFO[0m[0002] Downloading base image node:9-slim           
2019/05/02 04:57:39 No matching credentials were found, falling back on anonymous
[36mINFO[0m[0004] Executing 0 build triggers                   
[36mINFO[0m[0004] cmd: EXPOSE                                  
[36mINFO[0m[0004] Adding exposed port: 8080/tcp                
[36mINFO[0m[0004] Using files from context: [/workspace/source] 
[36mINFO[0m[0004] Skipping unpacking as no commands require it. 
[36mINFO[0m[0004] Taking snapshot of full filesystem...        
[36mINFO[0m[0004] ENV PORT 8080                                
[36mINFO[0m[0004] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0004] EXPOSE 8080                                  
[36mINFO[0m[0004] cmd: EXPOSE                                  
[36mINFO[0m[0004] Adding exposed port: 8080/tcp                
[36mINFO[0m[0004] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0004] WORKDIR /usr/src/app                         
[36mINFO[0m[0004] cmd: workdir                                 
[36mINFO[0m[0004] Changed working directory to /usr/src/app    
[36mINFO[0m[0004] Creating directory /usr/src/app              
[36mINFO[0m[0004] Taking snapshot of files...                  
[36mINFO[0m[0004] Using files from context: [/workspace/source] 
[36mINFO[0m[0004] COPY . .                                     
[36mINFO[0m[0004] Taking snapshot of files...                  
[36mINFO[0m[0004] CMD ["npm", "start"]                         
[36mINFO[0m[0004] No files changed in this command, skipping snapshotting. 
2019/05/02 04:57:42 existing blob: sha256:5bba3ecb4cd6374a19da45f2a3db0a9c0804984358b3bb4b666c28944016e5ff
2019/05/02 04:57:42 existing blob: sha256:7d083412657baa2dafba8d093034ef26f9ed2c90fb6ed95f76bb55aff03ed429
2019/05/02 04:57:42 existing blob: sha256:196b8e3c919d5639193ba6c6e54774528942041bbc37606631a2250502985c3c
2019/05/02 04:57:42 existing blob: sha256:e01bd28a434b56ffd2a3fe9aa99b9b69aecca69904843edd30b1ee7150d79904
2019/05/02 04:57:42 existing blob: sha256:a88538b9cd05cfa611e43a25582a95ad7879f6fa708b0386a26b71ee62235bb3
2019/05/02 04:57:46 pushed blob: sha256:30a7c9ad1589fce2c05e0ae081a22d852c4f78e04c4e25bf3fac72308b8c2e99
2019/05/02 04:57:46 pushed blob: sha256:be83e32bb2990060646f203c82721cba4a53c436c0ae55786ee895c23a004ec5
2019/05/02 04:57:46 pushed blob: sha256:a9003093904eebd44b49c0e3ad59c0fa40459f9d3d53e8fff9a9e84cdd13f6b8
2019/05/02 04:57:49 gcr.io/core-ui-dev/test-qs:0.0.19: digest: sha256:9c93b946d6d6454ee0c9ee0ad98dca0786c004cc2c76605c9b8e826b2088e17c size: 1404
Step: build-step-build-post-build:

no CVE provider running in the current ** namespace so skip adding image to be analysed
Step: build-step-promote-changelog:

Converted /workspace/source/charts/test-qs to an unshallow repository
Generating change log from git ref e109a35e0719abef14c078116c059e9b8884dd6a => f3b76dc321dcb85bf2c0c33be28620c014e1c0d2
WARNING: Failed to enrich commits with issues: User.jenkins.io "" is invalid: metadata.name: Required value: name or generateName is required
WARNING: Failed to enrich commits with issues: User.jenkins.io "" is invalid: metadata.name: Required value: name or generateName is required
Finding issues in commit messages using git format
WARNING: No release found for *********/test-qs and tag v0.0.19 so creating a new release
Updated the release information at https://github.com/*********/test-qs/releases/tag/v0.0.19
generated: /workspace/source/charts/test-qs/templates/release.yaml
Created Release test-qs-0-0-19 resource in namespace **
Updating PipelineActivity *********-test-qs-master-18 with version 0.0.19
Updated PipelineActivities *********-test-qs-master-18 with release notes URL: https://github.com/*********/test-qs/releases/tag/v0.0.19
Step: build-step-promote-helm-release:

WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Adding missing Helm repo: chartmuseum.jenkins-x.io http://chartmuseum.jenkins-x.io
Successfully added Helm repository chartmuseum.jenkins-x.io.
Adding missing Helm repo: jenkins-x-chartmuseum http://jenkins-x-chartmuseum:8080
Successfully added Helm repository jenkins-x-chartmuseum.
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Uploading chart file test-qs-0.0.19.tgz to http://jenkins-x-chartmuseum:8080/api/charts
Received 201 response: {"saved":true}
Step: build-step-promote-**-promote:

WARNING: prow based install so skip waiting for the merge of Pull Requests to go green as currently there is an issue with gettingstatuses from the PR, see https://github.com/jenkins-x/**/issues/2410
Promoting app test-qs version 0.0.19 to namespace **-staging
Created Pull Request: https://github.com/*********/environment-ivan-dev-staging/pull/22

WARNING: Failed to merge the Pull Request https://github.com/*********/environment-ivan-dev-staging/pull/22 due to PUT https://api.github.com/repos/*********/environment-ivan-dev-staging/pulls/22/merge: 405 Pull Request is not mergeable [] maybe I don't have karma?
Rebasing PullRequest due to conflict
Created Pull Request: https://github.com/*********/environment-ivan-dev-staging/pull/25

Pull Request https://github.com/*********/environment-ivan-dev-staging/pull/25 is merged at sha f86aa559515a2d4d6ab3fd4e59d255dc381c1e6f
Pull Request merged but we are not waiting for the update pipeline to complete!
